i18n/en.json

```diff
@@ -263,6 +263,14 @@
     "id": "api.channel.delete_channel.type.invalid",
     "translation": "Unable to delete direct or group message channels"
   },
+  {
+    "id": "api.channel.undelete_channel.unarchived",
```

Comment:
The translations are ordert by name so this has to be after `api.channel.rename_channel.cant_rename_group_messages.app_error`.

Reply:
done